plugins/dev-tools/src/toolboxCategories.js

```diff
@@ -771,84 +771,6 @@ export default {
         },
       ],
     },
-    {
-      kind: 'category',
```

Comment:
Oof this is making me realize I probably need to go an update some devsite screenshots... blehhh